lms/resources/_js_config/__init__.py

```diff
@@ -538,3 +539,23 @@ def _sync_api(self):
             return self._blackboard_sync_api()
 
         return None
+
+    def _add_deeplinking_api(self):
```

Comment:
What is deeplinking? Why is it added as part of `enable_content_item_selection_mode`? Comments and doc-strings would help.

Is this always canvas only, or are we just enabling it for canvas for the moment? What part does it play in a Canvas launch?

Reply:
I do know why we need to support deeplinking on LTI1.3 (to behave the same in both LTI versions) but I don't have a good answer to why we only use it canvas in LTI1.1.

I think the real reason might be that the first implementation supporting only canvas did and that defined what's expected of the integration there.